heka/cmd/heka-s3cat/main.go

```diff
@@ -53,6 +54,16 @@ func main() {
 		os.Exit(8)
 	}
 
+	workers := 1
+	if *flagWorkers == 0 {
+		fmt.Printf("Cannot run with zero workers. Using 1.\n")
+	} else if *flagWorkers < math.MaxUint32 {
```

Comment:
That is a lot of worker may want to cap it in the hundreds instead of the billions